h/feeds/render.py

```diff
@@ -40,8 +38,10 @@ def annotation_api_url(annotation):
         annotation_url=annotation_url, annotation_api_url=annotation_api_url,
         html_url=html_url, title=title, subtitle=subtitle)
 
-    return renderers.render_to_response(
+    response = renderers.render_to_response(
         'h:templates/atom.xml.jinja2', {"feed": feed}, request=request)
+    response.content_type = "application/atom+xml"
+    return response
```

Comment:
Huh. I wonder if we could add a failing / passing test for this? Seems like it'd just have to assert against the returned `response.content_type`?